src/modules/mavlink/mavlink_main.h

```diff
@@ -601,6 +605,9 @@ class Mavlink
 	uint8_t _network_buf[MAVLINK_MAX_PACKET_LEN];
 	unsigned _network_buf_len;
 #endif
+
+	const char *_mavlink_wifi_name;
```

Comment:
Can you rename this to `_interface_name` and default to nullptr, so that the previous behavior is unchanged and `__DEFAULT_MAVLINK_WIFI` is not needed?

Reply:
yes, did so